bin/gpodder

```diff
@@ -95,21 +95,34 @@ def main():
 
     parser = OptionParser(usage=s_usage, version=s_version)
 
-    parser.add_option("-v", "--verbose",
-                      action="store_true", dest="verbose", default=False,
-                      help=_("print logging output on the console"))
+    grp_subscriptions = OptionGroup(parser, "Subscriptions")
+    parser.add_option_group(grp_subscriptions)
 
-    parser.add_option("-q", "--quiet",
-                      action="store_true", dest="quiet", default=False,
-                      help=_("reduce warnings on the console"))
+    grp_subscriptions.add_option('-s', '--subscribe', dest='subscribe',
+                                 metavar='URL',
+                                 help=_('subscribe to the feed at URL'))
 
-    parser.add_option('-s', '--subscribe', dest='subscribe', metavar='URL',
-                      help=_('subscribe to the feed at URL'))
+    grp_logging = OptionGroup(parser, "Logging")
+    parser.add_option_group(grp_logging)
+
+    grp_logging.add_option("-v", "--verbose",
+                           action="store_true", dest="verbose", default=False,
+                           help=_("print logging output on the console"))
+
+    grp_logging.add_option("-q", "--quiet",
+                           action="store_true", dest="quiet", default=False,
+                           help=_("reduce warnings on the console"))
+
+    advanced = OptionGroup(parser, "Advanced")
```

Comment:
Name the variable grp_advanced?

Reply:
Right, that seems more consistent. Done!